### ISSUE TITLE
"f list" from console throws exception

### ISSUE THREAD

**Issue by volitank** [ProSavage/SavageFactions#79]
**Describe the bug**
When in the console, and not in game, using the command "f list" throws an exception

**To Reproduce**
1. Go to 'console'
2. Issue command 'f list'
3. See error

**Expected behavior**
Command should output the list of factions

**Server Version:**
 - Spigot Version:  git-Spigot-140f654-b446cb5 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)
 - Factions Version: Factions v1.6.9.5-U0.2.1-RC-1.5-STABLE - This was also happening on the Factions-2018-12-29--07-51.jar

**Additional context**
```
[02:29:26 WARN]: Unexpected exception while parsing console command "f list"
org.bukkit.command.CommandException: Unhandled exception executing command 'f' in plugin Factions v1.6.9.5-U0.2.1-RC-1.5-STABLE
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot-1.13.2.jar:git-Spigot-140f654-b446cb5]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:139) ~[spigot-1.13.2.jar:git-Spigot-140f654-b446cb5]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:702) ~[spigot-1.13.2.jar:git-Spigot-140f654-b446cb5]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchServerCommand(CraftServer.java:687) [spigot-1.13.2.jar:git-Spigot-140f654-b446cb5]
        at net.minecraft.server.v1_13_R2.DedicatedServer.handleCommandQueue(DedicatedServer.java:459) [spigot-1.13.2.jar:git-Spigot-140f654-b446cb5]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:418) [spigot-1.13.2.jar:git-Spigot-140f654-b446cb5]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835) [spigot-1.13.2.jar:git-Spigot-140f654-b446cb5]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733) [spigot-1.13.2.jar:git-Spigot-140f654-b446cb5]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.lang.NullPointerException
        at com.massivecraft.factions.cmd.CmdList.perform(CmdList.java:55) ~[?:?]
        at com.massivecraft.factions.zcore.MCommand.execute(MCommand.java:127) ~[?:?]
        at com.massivecraft.factions.cmd.FCommand.execute(FCommand.java:63) ~[?:?]
        at com.massivecraft.factions.zcore.MCommand.execute(MCommand.java:113) ~[?:?]
        at com.massivecraft.factions.cmd.FCommand.execute(FCommand.java:63) ~[?:?]
        at com.massivecraft.factions.zcore.MCommand.execute(MCommand.java:131) ~[?:?]
        at com.massivecraft.factions.zcore.MPlugin.handleCommand(MPlugin.java:288) ~[?:?]
        at com.massivecraft.factions.zcore.MPlugin.handleCommand(MPlugin.java:251) ~[?:?]
        at com.massivecraft.factions.SavageFactions.handleCommand(SavageFactions.java:494) ~[?:?]
        at com.massivecraft.factions.SavageFactions.onCommand(SavageFactions.java:505) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot-1.13.2.jar:git-Spigot-140f654-b446cb5]
        ... 8 more

**Comment by Driftay** [comment]
@volitank is the error ONLY when console performs the command?

**Comment by GenialJerome** [comment]
Fix in pull requests #86

**Comment by prosavage** [comment]
@GenialJerome fixed it in his recent pull request.
